invenio_app_ils/schemas/documents/document-v1.0.0.json

```diff
@@ -160,19 +123,466 @@
         }
       }
     },
-    "title": {
+    "abstracts": {
+      "items": [
+        {
+          "additionalProperties": false,
+          "properties": {
+            "source": {
+              "minLength": 1,
+              "type": "string"
+            },
+            "value": {
+              "minLength": 1,
+              "type": "string"
+            }
+          },
+          "required": ["value"],
+          "type": "object"
+        }
+      ],
+      "minItems": 1,
+      "title": "List of abstracts",
+      "type": "array",
+      "uniqueItems": true
+    },
+    "acquisition_source": {
+      "additionalProperties": false,
+      "properties": {
+        "submitter": {
+          "description": "This field is used only when the method is submitter",
+          "email": {
+            "format": "email",
+            "minLength": 4,
+            "title": "Email address of the submitter",
+            "type": "string"
+          },
+          "user_id": {
+            "title": "User ID of the submitter",
+            "type": "integer"
+          }
+        },
+        "method": {
+          "enum": ["batchuploader", "oai", "submitter"],
+          "minLength": 1,
+          "title": "Data obtaining method",
+          "type": "string"
+        },
+        "source": {
+          "minLength": 1,
+          "type": "string"
+        }
+      },
+      "title": "Origin of the metadata in the record",
+      "type": "object"
+    },
+    "alternative_titles": {
```

Comment:
Any reason why we don't group `alternative_titles` together with the others in the `titles` object?